src/canopy/knowledge_base/security_scanner/firewall.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import click
+import requests
+
+
+class AIFirewallError(ValueError):
+    pass
+
+
+class AIFirewall:
```

Comment:
Does RI have some kind of Python SDK? would be better to use it

Reply:
We do have a python SDK! However, it's a rather large package right now due to it supporting both of our products (Validation platform and AI Firewall), and so there were dependency conflicts with your existing dependencies. Since I didn't want to bloat your project, potentially complicate your future dev work with our dependency constraints, and also we're only making one type of call to the Firewall here, I thought it easier to just use the REST API. What do you think, happy to discuss